odoo/addons/base/models/ir_actions.py

```diff
@@ -87,24 +106,40 @@ def _get_eval_context(self, action=None):
 
     @api.model
     def get_bindings(self, model_name):
-        return self._get_bindings(model_name, bool(request) and request.session.debug)
-
-    @tools.ormcache('frozenset(self.env.user.groups_id.ids)', 'model_name', 'debug')
-    def _get_bindings(self, model_name, debug=False):
         """ Retrieve the list of actions bound to the given model.
 
            :return: a dict mapping binding types to a list of dict describing
                     actions, where the latter is given by calling the method
                     ``read`` on the action record.
         """
+        result = {}
+        for action_type, all_actions in self._get_bindings(model_name, bool(request)).items():
```

Comment:
`debug` seems broken. The wrong information gets sent to `_get_bindings`, which has stopped using the information as well as caching on it.

Why is the parameter still there, and why are we passing nonsense to it?

Reply:
Yes indeed, I am removing it.

It's no longer needed because `user_has_groups`/`has_group` correctly handles debug/`base.group_no_one` by default.